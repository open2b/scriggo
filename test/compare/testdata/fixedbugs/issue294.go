// paniccheck

package main

func f() {
	panic("f!")
}

func main() {
	f()
}
