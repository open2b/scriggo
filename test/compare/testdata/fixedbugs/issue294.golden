panic: panic 1: "f!"